pkg/services/contexthandler/contexthandler.go

```diff
@@ -235,7 +235,7 @@ func (h *ContextHandler) initContextWithAPIKey(reqContext *models.ReqContext) bo
 		return true
 	}
 
-	if apikey.ServiceAccountId < 1 { //There is no service account attached to the apikey
+	if apikey.ServiceAccountId == nil || *apikey.ServiceAccountId < 1 { //There is no service account attached to the apikey
```

Comment:
keeping the 2 conditions for retrocompatibility

Reply:
retrocompatibility? 🤔 what is this magic?